src/evict/evict_lru.c

```diff
@@ -1336,6 +1336,9 @@ __evict_lru_walk(WT_SESSION_IMPL *session)
              * don't exclude it.
              */
             queue->evict_candidates = 1 + candidates + ((entries - candidates) - 1) / 3;
+            if (queue->evict_candidates > entries / 2)
```

Comment:
Thanks @y123456yz for the PR. The change of picking more candidates is an intentional change that is done as part of this [commit](https://github.com/wiredtiger/wiredtiger/pull/2832/files#diff-fba173dbad7997f847117319e1939715f1969181ab2937508a2c9303b201f017). 

Do you find any problem with the earlier code that motivates you to change it? Or is it just a change from the code walk-through?

Reply:
hi, @kommiharibabu 
I'm just looking at the logic of the code and it doesn't make sense to restrict it.

Let me give you an example:
Example 1: entries=10000, candidates=5001, queue->evict_candidates=5001
Example 2: entries=10000, candidates=4999, queue->evict_candidates=1 + candidates + ((entries - candidates) -1) /3= 5000+5000/3=6667

It does not make sense that queue->evict_candidates in Example 2 have higher values than in example 1.

thanks.

Reply:
Thanks for the details. I agree with your description.